docs/index.md

```diff
@@ -6,26 +6,33 @@
 
 GreptimeDB is an open-source time-series database with a special focus on improving
 scalability, analytical capabilities, and efficiency. It's designed to work on
-infrastructure of the cloud era and benefits users from its elasticity and commodity
+the infrastructure of the cloud era, and users can benefit from its elasticity and commodity
 storage.
 
-Before getting started, please read the important documents below, including how to set up, fundamental concepts, architectural designs, and tutorials:
-
-- [Installation][1]: how to deploy GreptimeDB in either standalone or distributed mode
-
-- [Getting Started][2]: provides an overall introduction if you are new to Greptime DB
-
-- [User Guide][3]: for application developers to use GreptimeDB or build a custom integration
-
-- [Developer Guide][4]: for developers interested in learning more about the technical details and enhancing GreptimeDB as a contributor
-
-- [Changelog][5]: presents the latest GreptimeDB roadmap and biweekly reports
-
-- [FAQ][6]: presents the most frequently asked questions
-
-[1]: ./installation/overview.md
-[2]: ./getting-started/overview.md
-[3]: ./user-guide/concepts.md
-[4]: ./developer-guide/overview.md
-[5]: ./changelog/overview.md
-[6]: ./faq-and-others/faq.md
+Our core developers have been building time-series data platform for
+years, and based on their best practices, GreptimeDB was created to offer the following features:
+
+- [A standalone binary](https://github.com/GreptimeTeam/greptimedb/releases)
+  that can scale to a highly available distributed cluster, providing a transparent
+  experience for cluster users.
+- Optimized columnar layout for handling time-series data; compacted,
+  compressed, stored on various storage backends.
+- Flexible index options that address high cardinality issues.
+- Distributed, parallel query execution, leveraging elastic computing resource.
+- Native SQL, and Python scripting for advanced analytical scenarios.
```

Comment:
Native SQL, PromQL and Python

Reply:
Or we can use the latest description in readme https://github.com/GreptimeTeam/greptimedb